### PR TITLE
Update getting-started.md

### PR DESCRIPTION
**Pull request by pdmack**
Bump recommended kf version

/assign @jlewi 

Are we ready to change this? (i.e., all images tagged?)

### CODE REVIEW

File: content/docs/started/getting-started.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ kubectl create namespace ${NAMESPACE}
 # Which version of Kubeflow to use
 # For a list of releases refer to:
 # https://github.com/kubeflow/kubeflow/releases
-VERSION=v0.1.2
+VERSION=v0.2.0
 
 # Initialize a ksonnet app. Set the namespace for it's default environment.
 APP_NAME=my-kubeflow
```
Comment (jlewi) [inline content/docs/started/getting-started.md]: Thanks

Can you please remove the tf-job package and add the examples package?

**Comment by pdmack** [comment]
@jlewi PTAL

File: content/docs/started/getting-started.md [hunk 2]
```diff
@@ -95,7 +95,7 @@ ks registry add kubeflow github.com/kubeflow/kubeflow/tree/${VERSION}/kubeflow
 
 ks pkg install kubeflow/core@${VERSION}
 ks pkg install kubeflow/tf-serving@${VERSION}
-ks pkg install kubeflow/tf-job@${VERSION}
+ks pkg install kubeflow/examples/tf-job-simple@${VERSION}
```
Comment (jlewi) [inline content/docs/started/getting-started.md]: ks pkg install kubeflow/examples@${VERSION}

Comment (pdmack) [reply content/docs/started/getting-started.md]: whatever!

**Pull request merged**